### PR TITLE
Added cyclic sort

### PR DESCRIPTION
**Pull request by Poobalan1210**
Added the python code for cyclic sort.

### CODE REVIEW

File: python/sorting/cyclic_sort.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            # Put the item there or right after any duplicates.
+            while item == num_list[pos]:
+                pos += 1
+            num_list[pos], item = item, num_list[pos]
+            writes += 1
+ 
+    return writes
+ 
+ 
+num_list = [75, 16, 55, 19, 48, 14, 2, 61, 22, 100]
```
Comment (geekquad) [inline python/sorting/cyclic_sort.py:42]: Please make it a user-driven program. The user should give the inputs.

**Review by geekquad — CHANGES_REQUESTED** [review]
Please make the necessary changes.

File: python/sorting/cyclic_sort.py (hunk 1 above)
Comment (Poobalan1210) [reply python/sorting/cyclic_sort.py:42]: Ok

**Comment by Poobalan1210** [comment]
I have updated the code to get user input.

File: python/sorting/cyclic_sort.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            num_list[pos], item = item, num_list[pos]
+            writes += 1
+ 
+    return writes
+
+if __name__ =="__main__":
+    input_array=input("Enter the elements seperated by space ").strip()
+    num_list=[int(num) for num in input_array.split()]
+    #prints the writes
+    print(cycle_sort(num_list)
```
Comment (geekquad) [inline python/sorting/cyclic_sort.py:45]: syntax error

File: python/sorting/cyclic_sort.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    #prints the sorted list
+    print(num_list)
```
Comment (geekquad) [inline python/sorting/cyclic_sort.py:47]: syntax error

**Review by geekquad — CHANGES_REQUESTED** [review]
The program is not giving the desired output. Please compile it before making any PR. Thankyou.

**Comment by Poobalan1210** [comment]
Syntax error has been resolved.

**Review by geekquad — APPROVED** [review]

**Comment by Poobalan1210** [comment]
Should I make some other changes to merge the PR.

**Comment by geekquad** [comment]
No @Poobalan1210. Already merged.

**Pull request merged**